app/javascript/components/public_health/actions/UpdateCaseStatus.js

```diff
@@ -223,6 +239,7 @@ UpdateCaseStatus.propTypes = {
   authenticity_token: PropTypes.string,
   patients: PropTypes.array,
   close: PropTypes.func,
+  monitoring_reasons: PropTypes.array,
 };
 
 export default UpdateCaseStatus;
```

Comment:
One issue with this bulk action (it's an edge case but probably should be fixed).

**Step to reproduce:**
1. select a patient on the line list on the PH dashboard
2. open the patient show page in a new tab (for easier access in step 8)
3. open the case status bulk action
4. select "confirmed" or "probable" for case status
5. select "end monitoring" for the action
6. select a reason such as "does not meet criteria for monitoring"
7. change the case status to something other than "confirmed" or "probable" (and not what it's currently set to for the patient)
8. go to the patient's show page
9. see that the history item for changing case status incorrectly included the reason for closure ("does not meet criteria for monitoring")

**Example of the incorrect history item:**
![Screen Shot 2021-05-05 at 12 48 38 PM](https://user-images.githubusercontent.com/24628687/117193547-388bf100-ada0-11eb-859f-21ce44270be6.png)

Reply:
Fixed in latest commit! Good catch!